gfx/Mushroom-Dream/pngs_tall_32x48/terrain/t_shrub/t_shrub.txt

```diff
@@ -0,0 +1,8 @@
+  {
+    "id": "t_shrub",
+    "fg": [
+	{ "weight": 70, "sprite": "t_shrub1" },
+	{ "weight": 30, "sprite": "t_shrub2" }
+	],
+    "bg": ""
```

Comment:
You need to add some background here or it will just show black

Reply:
yeah i noticed, i'm trying to do it now xD

Reply:
btw what is the tag for harvested shrubs and non harvested? are the ones in there correct?

Reply:
Yep it's the right one. Harvested shrubs also need a background otherwise the dirt below the shrub will desapear when you harvest it X)

Reply:
i'll add that right now

Reply:
done 👍